app/src/main/java/protect/card_locker/AboutActivity.java

```diff
@@ -62,6 +62,7 @@ protected void onCreate(Bundle savedInstanceState) {
         }
 
         final List<ThirdPartyInfo> USED_LIBRARIES = new ArrayList<>();
+        USED_LIBRARIES.add(new ThirdPartyInfo("uCrop","https://github.com/Yalantis/uCrop.","Apache 2.0"));
```

Comment:
There's an extra . at the end of the URL that shouldn't be there

Reply:
removed :)